index.html

```diff
@@ -1033,6 +1034,17 @@
       </p>
       <dl>
         <dt>
+          <dfn>paymentRequestId</dfn>
+        </dt>
+        <dd>
+          <a>paymentRequestId</a> is a freeform identifier for this
+          payment request. This field is intended to hold a unique
+          identifier for the purposes of recovering from failures and/or as a
+          foreign key into other systems. If a <a>paymentRequestId</a>
+          is not provided by the client then a UUID will be generated and stored
```

Comment:
The third sentence should be a non-normative note. Also, there is no requirement that this shall be a UUID in the payment request constructor.